Assignment_display_marks/Student.php

```diff
@@ -0,0 +1,107 @@
+<?php
+  /**
+   * Student class to store student operations
+   */
+  class Student
+  {
+    public $student;
+    public $subject;
+
+    function __construct($student,$subject)
+    {
+      $this->student = $student;
+      $this->subject = $subject;
+    }
+    public function get_data()
+    {
+      // $this->show_details($this->student);
+      // $this->show_details($this->subject);
+    }
+    public function subject_fetch($grade){
+      echo "<br>Subject : <br>";
+      $flag = 0;
+      foreach ($this->subject as $key => $value){
+        if ($grade == $key) {
+          $flag = 1;
+          for ($i=0; $i < sizeof($value); $i++){
+            foreach ($value[$i] as $k => $v) {
+              echo $k." : ".$v."<br>";
+            }
+          }
+        }
+      }
+      if ($flag == 0) {
+        echo "<br>Not found grade!!";
+      }
+    }
+    public function student_marks($id){
+      $flag = 0;
+      echo "<br>Marks list : <br>";
+      for ($i=0; $i < sizeof($this->student); $i++) {
+        if ($id == $this->student[$i]['id']) {
+          $flag = 1;
+          foreach ($this->student[$i]['marks'] as $key => $value) {
+            echo $key." : ".$value."<br>";
+          }
+        }
+      }
+      if ($flag == 0) {
+        echo "<br>Details not found!!!";
+      }
+    }
+    public function result(){
```

Comment:
Same here, return the result for the table in an array and display it on the index.php